lib/wallet/rpc.js

```diff
@@ -2308,6 +2280,34 @@ class RPC extends RPCBase {
     };
   }
 
+  _validateUpdateItempotent(args, help, method) {
```

Comment:
Typo: Itempotent

Reply:
I copy-pasted this from the other function, so that must be wrong too